src/LeanCode.Contracts.Admin/AdminQuery.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace LeanCode.Contracts.Admin;
+
+public abstract class AdminQuery<TResult> : IQuery<AdminQueryResult<TResult>>
+{
+    public int Page { get; set; }
+    public int PageSize { get; set; }
+
+    public SortOrderDTO? SortOrder { get; set; }
+    public string? SortBy { get; set; }
+}
+
+public enum SortOrderDTO
+{
+    Descending = 0,
+    Ascending = 1

```

Comment:
Can we invert this so the default is ascending? Seems more natural to me.

(But tbh I'd strongly prefer to instead have `bool Descending` property in the `AdminQuery` instead.)

Reply:
I agree, but maybe "**Sort**Descending"?